README.md

```diff
@@ -1,7 +1,8 @@
 # cna-installer
+
 ```bash
 cna-installer is a binary that installs, sets-up and configures a
-kubernetes cluster with the CNA stack applications.
+Kubernetes cluster with the CNA stack applications.
```

Comment:
I think this is a false-positive, since it's attempting to capitalise a word in mid-sentence.

Reply:
Not sure - I believe you should always write "Kubernetes cluster" with capital "K". For example: https://kubernetes.io/docs/setup/

Reply:
got it :) lgtm, then.
you might want to fix the command output then in https://github.com/trawler/cna-installer/blob/ff741bdb2b4b6e33474fd213c019ee1486bc0473/cmd/main.go#L29 ;)

Reply:
Done...